app/src/main/java/io/getstream/chat/android/core/poc/library/ChatApiImpl.kt

```diff
@@ -1,21 +1,32 @@
 package io.getstream.chat.android.core.poc.library
 
+import com.google.gson.Gson
+import io.getstream.chat.android.core.poc.app.App
 import io.getstream.chat.android.core.poc.library.api.QueryChannelsResponse
 import io.getstream.chat.android.core.poc.library.call.ChatCall
 import io.getstream.chat.android.core.poc.library.gson.JsonParser
+import io.getstream.chat.android.core.poc.library.requests.QueryUsers
 import io.getstream.chat.android.core.poc.library.rest.*
 import io.getstream.chat.android.core.poc.library.socket.ConnectionData
+import java.util.*
 
 class ChatApiImpl(
     private val apiKey: String,
     private val retrofitApi: RetrofitApi,
-    private val jsonParser: JsonParser
+    private val jsonParser: JsonParser,
+    private val application: App
```

Comment:
There should not be `App`. `App` is just testing code, it will be removed from library.